frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -5,7 +5,7 @@
     </div>
     <b-card no-body>
       <b-tabs
-        class="sidebar"
+        class="sidebar introjs-time"
```

Comment:
Parece que el lugar indicado para agregar clases al sidebar es abajo en la línea 12 donde aparece `nav-wrapper-class`
```suggestion
        class="sidebar"
```